File: Iniciante-Begginer/Python/#1101 - #1150/#1132-Multiplos13.py
x = int(input())
y = int(input())
soma = 0

if y > x:
    for i in range(x, y+1):
        if i % 13 == 0:
            continue
        else:
            soma = soma + i
else:
    for i in range(y, x+1):
        if i % 13 == 0:
            continue
        else:
            soma = soma + i
print(soma)

File: Iniciante-Begginer/C/#1101 - #1150/#1132-Multiplos13.c
#include <stdio.h>
 
int main() {
    int x, y, soma = 0;

    scanf("%d %d",&x, &y);
    
    if ( y > x){
        for(x; x <= y; x++){
            if( x % 13 != 0){
                soma += x;
            }
        }
    }else{
        for(y; y <= x; y++){
            if( y % 13 != 0){
                soma += y;
            }
        }
    }
    printf("%d\n",soma);
    return 0;
}